hydrolib/core/io/obscrosssection/models.py

```diff
@@ -0,0 +1,86 @@
+from typing import List, Literal, Optional
+
+from pydantic.fields import Field
+
+from hydrolib.core.io.ini.models import INIBasedModel, INIGeneral, INIModel
+from hydrolib.core.io.ini.util import (
+    get_location_specification_rootvalidator,
+    get_number_of_coordinates_validator,
+    get_split_string_on_delimiter_validator,
+)
```

Comment:
A general note: it's not an observation _point_ cross section, but an observation cross section. Please update the names and the comments.

Reply:
Good _point_. Fixed.